gateway/1.4/modules/ROOT/pages/_partials/logging-severity-level.adoc

```diff
@@ -0,0 +1,67 @@
+[[severity-levels]]
+== Message Logging Severity Levels
+
+When you configure a Message Logging policy for your API instance you can assign a severity level. 
+
+=== Severity Levels Description
+
+Each severity level includes a specific set of information in the log report. The following table
+describes the type of information that will be included in the logs for each severity level.
+
+[%header,cols='1a,5a'] 
+|===
+|Log Severity Level
+|Description
+
+|ERROR
+|Error events that prevent normal program execution but might allow the application to continue running.
+
+|WARN
+|Potentially harmful situations that indicate potential problems.
+
+|INFO
+|Informational messages that highlight the progress of the application.
+
+|DEBUG
+|Tracing information used by application developers.
+
+|===
+
+=== Severity Levels Hierarchy
+
+Log reports include only the severity level you select and the subsequent levels on the hierarchy .
+For example, if you select INFO, the log reports include the WARN and ERROR severity levels.
+However, if you select ERROR, the log reports include only error messages. The following table describes the hierarchy of the different severity levels:
```

Comment:
Oh, I totally misinterpreted. Is there a similar log-level description elsewhere that you can poach? For CH2, each log level is selected individually, eliminating this difficult-to-document scenario.

I wonder if putting them from least to most severe. Then each level will include the subsequent levels.

Reply:
Oh, you have another table to describe this. I think if you order the first table:
DEBUG
INFO
WARN
ERROR

Then, each level includes the levels that follow. And you can delete the hierarchy table.